src/components/layout/FixedLeft/FixedLeft.jsx

```diff
@@ -26,8 +26,8 @@ type State = {|
 |};
 
 type Props = {|
-  children: Array<Node>,
-  stickAt: number,
+  children: React$Node,
```

Comment:
I think `children` can be optional, in general.

Reply:
I did not want to change tons of API in this PR only what was needed